mkt/ratings/views.py

```diff
@@ -100,14 +100,13 @@ def get_app(self, ident):
         except Webapp.DoesNotExist:
             raise Http404
         current_region = get_region()
-        if ((not app.is_public() or
-             not app.listed_in(region=current_region)) and
+        if (not app.is_public() and
                 not check_addon_ownership(self.request, app)):
```

Comment:
yeah, I'm suggesting 2 lines by going to the next line after `check_addon_ownership(`